dev_tools/conf/pip-list-dev-tools.txt

```diff
@@ -37,8 +35,5 @@ nbsphinx
 ipython
 ipykernel
 
-# For google notebooks
-google-colab
```

Comment:
Will this break the Colabs that have engine auth examples?

Reply:
No it won't on colab. 
On jupyter locally we have issues anyways because of exactly google-colab: #3318 
On jupyter I'd suggest using something lighterweight, e.g. this works very nicely: 

```
# The Google Cloud Project id to use.
project_id = 'your project' #@param {type:"string"}


def setup_auth():
  """Runs the user through the Colab OAuth process.
  
  Sets the local Application Default Credentials. For more information on on 
  using Application Default Credentials see 
  https://cloud.google.com/docs/authentication/production
  """
  import pydata_google_auth
  credentials = pydata_google_auth.get_user_credentials(
    ['https://www.googleapis.com/auth/cloud-platform'],
  )


print("Getting OAuth2 credentials.")
print("Press enter after entering the verification code.")
setup_auth()
print("Authentication complete.")

try: 
    import cirq 
except:     
    !pip install cirq

print("Cirq is installed.")
```

I think I'll just update our colab/notebooks with this new header.

Reply:
Yeah, I guess that's okay then.  I think I added it since readthedocs needed this for one of the notebooks.  If docs build and you can run it in colab, I guess it's fine, assuming we switch to that lightweight alternative.

Reply:
Yeah, I remember that issue - specifically this build failed because of lack of the dependency on your first PR introducing these colabs: https://github.com/quantumlib/Cirq/runs/1052329100

I'm not sure what changed besides 3.6 -> 3.7 in the CI - but that doesn't explain why it works on RTD which still builds it on 3.6...
It is building now both on our CI and in RTD: https://balopat-cirq-rtd.readthedocs.io/en/latest/docs/tutorials/google/start.html (https://readthedocs.org/projects/balopat-cirq-rtd/builds/11997856/). I'm genuinely puzzled why the error happened on your branch then - I don't see any changes that would explain why it went away. 

Added #3368 for the lighter alternative.

Reply:
I'm going to go ahead and merge. If something goes wrong, I will fix it.